.github/workflows/ci.yml

```diff
@@ -41,6 +41,19 @@ jobs:
           name: logs
           path: test/server.log
 
+  index_resources:
```

Comment:
_non-blocking, minor nit_

We tend to use hyphens in our job names (and workflow names), not underscores.

Reply:
Updated to hyphens in https://github.com/nextstrain/nextstrain.org/pull/841/commits/cd51343ad370085e39f25f738318962bab328f61.